Tools/Scripts/hooks/prepare-commit-msg

```diff
@@ -121,7 +120,16 @@ def message(source=None, sha=None):
 
     bugs_string = get_bugs_string()
 
-    return '''{title}
+    revert_msg = os.environ.get('COMMIT_MESSAGE_REVERT', '')
+    if revert_msg:
+        return '''{title}\n{revert}\n{bugs}
+    '''.format(
```

Comment:
I think order is wrong here. Why are `bugs` coming after the revert message? Also, you're already using `\n` in this string, I think this string should be a single line.

Reply:
It's because the `get_bugs_string()` is going to get the bugs for the commit that is being reverted, not the new revert bug. So that one needs to go under the "Reverted changeset:" line which is set in `revert`.

Reply:
Got it!